CMakeLists.txt

```diff
@@ -70,8 +70,8 @@ endif()
 set(WITH_LFORTRAN_ASSERT ${WITH_LFORTRAN_ASSERT_DEFAULT}
```

Comment:
Some of the variables like `WITH_LFORTRAN_ASSERT` are connected to libasr, so I decided not to modify it (as it also affects LFortran)